content/en/blog/2023/ambient-merged-istio-main/index.md

```diff
@@ -19,7 +19,7 @@ Ambient mesh is designed for simplified operations, broader application compatib
 
 ## Get involved
 
-Follow our [getting started guide](http://istio.io/latest/docs/ops/ambient/getting-started/) to try the ambient pre-alpha build today. We'd love to hear from you! To learn more about ambient:
+Follow our [getting started guide](http://istio.io/latest/docs/ambient/getting-started/) to try the ambient pre-alpha build today. We'd love to hear from you! To learn more about ambient:
```

Comment:
Should use an absolute link instead of external:
```suggestion
Follow our [getting started guide](/docs/ambient/getting-started/) to try the ambient pre-alpha build today. We'd love to hear from you! To learn more about ambient:
```